util/cron/test-apptainer.bash

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+#
+# This script will build and run chapel in different linux distros using apptainer. 
+# For detailed info please refer README located at util/deval/test/apptainer/README.md
+
+CWD=$(cd $(dirname $0) ; pwd)
+source $CWD/common.bash
+
+export CHPL_HOME=$(cd $CWD/../.. ; pwd)
+log_info "Setting CHPL_HOME to: ${CHPL_HOME}"
+
+cd $CHPL_HOME/util/devel/test/apptainer
+
+./chapel-quickstart.sh
+
+if [ $? -ne 0 ] 
+   then
+     log_error "./chapel-quickstart.sh exited with error" 
+     exit 1
+   else
+     log_info "./chapel-quickstart.sh succeeded"  
+   fi 
+
+./chapel-default.sh 
```

Comment:
ok, I will run ./chapel-default.sh and get the list of failures to figure out what failures can be ignored.

Reply:
Since ./chapel-quickstart.sh takes almost 12 hours to complete,  Can we add the check in a 2 step process?

1.  Get this script checked in (fixing just the typo) and create a jenkins job that will start this test run( The test run will not send email to discourse until step 2 is completed) on the VM and wait for the results.

2. Based on the results, add a logic to ignore the build failures on some platforms. Once that is working, fix jenkins config to send the email to discourse.

Reply:
I'm pretty sure we already know which platforms will fail -- any `-nollvm` configurations as well as those I covered in my email "1.30 portability testing".

FWIW, I was expecting your initial PR to only add quickstart testing. All the quickstart configurations should pass today. So, if we wanted something to pass initially, you could just comment out the chapel-default.sh part in this PR for now & merge that way. It is also OK with me if you want to see the failure in jenkins & you proceed with the plan you described.

Reply:
Thanks Michael. 
Yes, all quickstart configs passed.
I will send a patch with chapel-default.sh commented out.